gabbi/handlers/jsonhandler.py

```diff
@@ -54,7 +55,7 @@ def dumps(data, pretty=False, test=None):
 
     @staticmethod
     def loads(data):
-        return json.loads(data)
+        return yaml.safe_load(data)
```

Comment:
The root of the speedbumps you've hit is because you're engaging `yaml.safe_load` here, instead of in the `action` method, meaning yaml gets involved in all the JSON processing, not just when you are loading data from disk to be handled in a `response_json_paths`.

This might be the right thing, but did you consider doing it in `action` and keeping the behaviors separated?